runtime/doc/diagnostic.txt

```diff
@@ -365,6 +365,10 @@ config({opts}, {namespace})                          *vim.diagnostic.config()*
                          the beginning of the virtual text.
                        • prefix: (string) Prepend diagnostic message with
                          prefix.
+                       • suffix: (string or function) Append diagnostic message
+                         with suffix. If a function, it must have the signature
+                         (diagnostic) -> string. This can be used to render an
```

Comment:
Add a link to `|diagnostic-structure|` somewhere in here so it's clear what the function argument is.